### PR TITLE
Proper dev stack

### PR DESCRIPTION
**Pull request by clemsos**
# Description

Provide a working `docker-compose up --build`

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -5,7 +5,6 @@ In effort to aid the ease of development, we have provided a Docker compose file
 The following services are run as part of the compose setup:
 
 * eth-node (hardhat)
-* jaeger
```
Comment (julien51) [inline docker/README.md:8]: 🧹

File: docker/development/subgraph/startup.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+yarn workspace @unlock-protocol/subgraph codegen
+
+# build the subgraph files
+yarn workspace @unlock-protocol/subgraph build --network mainnet
+
+# init the graph
+yarn workspace @unlock-protocol/subgraph run create --network mainnet
+
+# deploy
+yarn workspace @unlock-protocol/subgraph run deploy --network mainnet --label 0.0.1
```
Comment (julien51) [inline docker/development/subgraph/startup.sh:18]: Wait, does that deplyon `mainnet` ?

File: docker/docker-compose.override.yml [hunk 3]
```diff
@@ -1,25 +1,14 @@
-version: '3.2'
+version: '3.4'
 services:
   
   eth-node:
     restart: always
     build:
       context: ./development/eth-node
+    command: [ "yarn", "start", "--hostname", "0.0.0.0"]
```
Comment (julien51) [inline docker/docker-compose.override.yml:8]: Maybe add the mnemonic so it always has the same accounts with some eth?

**Review by julien51 — APPROVED** [review]
LGTM!

**Comment by julien51** [comment]
Awesome, feel free to merge when you've done these things :)

File: docker/development/subgraph/startup.sh (hunk 2 above)
Comment (clemsos) [reply docker/development/subgraph/startup.sh:18]: no but subgraph config only support mainnet/rinkeby/ropsten as network names...

File: docker/docker-compose.override.yml (hunk 3 above)
Comment (clemsos) [reply docker/docker-compose.override.yml:8]: yep, added in the hardhat config file already

**Pull request merged**